content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -693,7 +675,84 @@ data:
 
 - You can't use ConfigMaps for {{< glossary_tooltip text="static pods" term_id="static-pod" >}}, because the Kubelet does not support this.
 
+### Optional ConfigMaps
+
+In a Pod, or pod template, you can mark a reference to a ConfigMap as _optional_.
```

Comment:
```suggestion
You can mark a reference to a ConfigMap as _optional_ in a Pod specification.
```

Reply:
Updated in https://github.com/kubernetes/website/pull/35160/commits/ad33b0c10757978ffaddfa66a691dc9ea302bec1